pkg/virt-controller/watch/migration_test.go

```diff
@@ -172,7 +173,7 @@ var _ = Describe("Migration watcher", func() {
 		config, _, _, _ := testutils.NewFakeClusterConfig(&k8sv1.ConfigMap{})
 
 		controller = NewMigrationController(
-			services.NewTemplateService("a", "b", "c", "d", "e", "f", "g", pvcInformer.GetStore(), virtClient, config, qemuGid),
+			services.NewTemplateService("a", "b", "c", "d", "e", "f", "g", pvcInformer.GetStore(), virtClient, config, qemuGid, runtime.GOARCH),
```

Comment:
Should this be a table driven test now where we pass in the different architectures?

Reply:
Yes, you are right

Reply:
> Should this be a table driven test now where we pass in the different architectures?

Hi Roman, I do some investigation on this, and have not get an idea howto imply it.
Usually we define a table to contain variable that may changed in testcases, then use a loop to run all test. However, in this test, NewMigrationController is defined in "BeforeEach", how can we use a loop to use two "BeforeEach".

Reply:
Lets keep it for now: I see more cases where we may want to increase our unit test coverage for more architectures. Right now the main and only supported architecture is unit-tested like it is done. Good enough for me on this PR. Let's optimize this on a followup.